src/components/reviewMatches/SearchModal.tsx

```diff
@@ -22,36 +23,47 @@ const options: ToggleButtonOptions[] = [
   { value: 1, label: 'Simple search' }
 ]
 
-const SearchModal: FC<{ isOpen: boolean; onClose: () => void }> = ({
-  isOpen,
-  onClose
-}) => {
-  const [tabValue, setTabValue] = useState('Custom search')
+const SearchModal: FC<{
+  isOpen: boolean
+  onClose: () => void
+  onChange: (query: SearchQuery | CustomSearchQuery | undefined) => void
+}> = ({ isOpen, onClose, onChange }) => {
+  const [selectedTab, setSelectedTab] = useState('Custom search')
```

Comment:
As I mentioned in my previous comment : 
```suggestion
  const [selectedTab, setSelectedTab] = useState(SEARCH_TYPE.customSearch)
```